src/server/raftkv.rs

```diff
@@ -202,6 +223,44 @@ impl<S: RaftStoreRouter<RocksSnapshot>> RaftKv<S> {
             .map_err(From::from)
     }
 
+    fn exec_write_batch(
```

Comment:
Prefer names like `exec_raw_write`, correspond to `send_raw_command`.

Reply:
OK.